shared/dates.ts

```diff
@@ -133,3 +133,11 @@ export const getOldestDate = (dates: Date[]) =>
 	dates.reduce((dateA: Date, dateB: Date) =>
 		dateA.valueOf() <= dateB.valueOf() ? dateA : dateB,
 	);
+
+export function convertTimestampToDate(timestamp: number): string {
+	const date = new Date(timestamp);
+	const day = date.getDate();
+	const month = date.toLocaleString('default', { month: 'short' });
+	const year = date.getFullYear();
+	return `${day} ${month} ${year}`;
```

Comment:
there is definitely a format string to do this, would `DATE_FNS_LONG_OUTPUT_FORMAT` not do the trick?

Reply:
you could do `dateString(new Date(timestamp), DATE_FNS_LONG_OUTPUT_FORMAT)` instead